packages/ember-glimmer/tests/integration/helpers/get-test.js

```diff
@@ -0,0 +1,337 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import { set } from 'ember-metal/property_set';
+import { get } from 'ember-metal/property_get';
+import EmberObject from 'ember-runtime/system/object';
+import TextField from 'ember-views/views/text_field';
+import Component from 'ember-views/components/component';
+
+moduleFor('Helpers test: {{get}}', class extends RenderingTest {
+  ['@test should be able to get an object value with a static key']() {
+    this.render(`[{{get colors 'apple'}}] [{{if true (get colors 'apple')}}]`, {
+      colors: { apple: 'red', banana: 'yellow' }
+    });
+
+    this.assertText('[red] [red]');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('[red] [red]');
+
+    this.runTask(() => set(this.context, 'colors', {
+      apple: 'green', banana: 'purple'
+    }));
+
+    this.assertText('[green] [green]');
+  }
+
+  ['@test should be able to get an object value with nested static key']() {
+    this.render(`[{{get colors "apple.gala"}}] [{{if true (get colors "apple.gala")}}]`, {
+      colors: { apple: { gala: 'red and yellow' }, banana: 'yellow' }
+    });
+
+    this.assertText('[red and yellow] [red and yellow]');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('[red and yellow] [red and yellow]');
+
+    this.runTask(() => set(this.context, 'colors', {
+      apple: { gala: 'yellow and red striped' }, banana: 'purple'
+    }));
+
+    this.assertText('[yellow and red striped] [yellow and red striped]');
+
+    this.runTask(() => set(this.context, 'colors.apple.gala', 'yellow-redish'));
+
+    this.assertText('[yellow-redish] [yellow-redish]');
+  }
+
+  ['@test should be able to get an object value with a bound/dynamic key']() {
+    this.render(`[{{get colors key}}] [{{if true (get colors key)}}]`, {
+      colors: { apple: 'red', banana: 'yellow' },
+      key: 'apple'
+    });
+    this.assertText('[red] [red]');
+
+    this.runTask(() => set(this.context, 'key', 'banana'));
+    this.assertText('[yellow] [yellow]');
+
+    this.runTask(() => set(this.context, 'colors', {
+      apple: 'green',
+      banana: 'purple'
+    }));
+    this.assertText('[purple] [purple]');
+
+    this.runTask(() => set(this.context, 'key', 'apple'));
+    this.assertText('[green] [green]');
+
+    this.runTask(() => set(this.context, 'colors.apple', 'red'));
+    this.assertText('[red] [red]');
```

Comment:
also – extremely nit: there are some inconsistency between blank lines `this.runTask` and `this.assertText`, sometimes they are bundled, sometimes there is a new line between them; I think we happen to prefer the moar space side of things in the new tests, but you can just pick one or the other for this file and stick to it.